app/helpers/confirmation-message-text.js

```diff
@@ -0,0 +1,15 @@
+import { helper } from '@ember/component/helper';
+import { assert } from '@ember/debug';
+
+const CONFIRMATION_MESSAGE_TEXT = {
+  nonActiveRelatedOrganization:
+    'Opgelet, je hebt een niet actieve organisatie geselecteerd. Ben je zeker dat je deze relatie wenst toe te voegen?',
+};
+
+export default helper(function confirmationMessageText([key]) {
```

Comment:
Wrapping functions in the helper util is no longer needed since ember-source 4.5+ (the blueprint was simply outdated for a very long time).

```suggestion
export default function confirmationMessageText(key) {
```

Reply:
> Wrapping functions in the helper util is no longer needed since ember-source 4.5+ (the blueprint was simply outdated for a very long time).
> 
> ```suggestion
> export default function confirmationMessageText(key) {
> ```

Thanks for point that out. Fixed it for this instance. Will probably take a stab at doing the same for other helpers at some point.